.github/workflows/_tests.yml

```diff
@@ -52,6 +52,7 @@ jobs:
     env:
       DOCKER_BUILDKIT: 1
       BUILDKIT_PROGRESS: plain
+      YARN_ENABLE_IMMUTABLE_INSTALLS: false
```

Comment:
Please explain why this is needed.

Reply:
ha yes indeed, please remove this change. Make sure you run `yarn` to update the yarn.lock file and commit the new yarn.lock file!